### PR TITLE
Change Dates Within the Transaction PDF to Intl Standard

### PR DESCRIPTION
**Pull request by SudharakaP**
This changes the dates within the transaction pdf to intl standard.

Fixes https://github.com/opencollective/opencollective/issues/3096

### CODE REVIEW

**Comment by znarf** [comment]
Can you give an example of how it was formatted before, and how it is formatted after? I'm not sure to understand.

**Comment by SudharakaP** [comment]
@znarf : The dates previously was in MM/DD/YYYY format within the file. Here is an example,

[result-before.pdf](https://github.com/opencollective/opencollective-pdf/files/4567709/result-before.pdf)

Now I have converted it to YYYY/MM/DD format. Example,

[result-after.pdf](https://github.com/opencollective/opencollective-pdf/files/4567713/result-after.pdf)

Now I wonder if we should do this for all the other pdf files as well; if so let me know I can change it for all. :smile:

**Comment by SudharakaP** [comment]
@Betree : I've tried to achieve this through configuration but wasn't successful. Therefore I extracted this to a separate component. Take a look and let me know your thoughts. :thinking: Maybe we should use this component everywhere to represent dates?

File: components/CustomIntlDate.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
+import { FormattedDateParts } from 'react-intl';
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const CustomIntlDate = ({ date }) => {
+  return (
+    <FormattedDateParts value={date} day="2-digit" month="2-digit" year="numeric">
+      {(parts) => <>{`${parts[4].value}/${parts[0].value}/${parts[2].value}`}</>}
```
Comment (Betree) [inline components/CustomIntlDate.js:8]:
```suggestion
      {(parts) => <>{`${parts[4].value}-${parts[0].value}-${parts[2].value}`}</>}
```

Comment (SudharakaP) [reply components/CustomIntlDate.js]: @Betree : Thanks. Done. :+1:

**Pull request merged**